index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <body style="background-color:lightgray;"></body>  
+  <link rel="stylesheet" href="css/style.css">
+  <head>
+    <h1 id= "title"> A Tribute to Micheal Jackson </h1>
+    <p>King of Pop</p>
+    
+    <main id= main>
+    <meta charset="UTF-8">
+    <title>Raf & Boku project</title>
+    
+  </head>
+  <figure id="img-div">
+<img id= "image" class="center" src="https://www.billboard.com/wp-content/uploads/stylus/2528869-michael-jackson-617-409.jpg?w=617&h=409&crop=1
+">
+<style type="text/css">
+    img {
+  border: 50px solid rgb(255, 255, 255);
+}
+    h1 {text-align: center;}
+    h2 {text-align: center;}
+    div {text-align: center;}
+    p {text-align: center;}
+        .center {
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    width: 70%;
+  }
+    </style>
+</figure>
+
+<div id="img-caption">
+  <strong>Michael Joseph Jackson</strong>
+  </div>
+  <body> <p>  <ul> 
+  <li>He was an American singer, songwriter, and dancer. </li>  <li>He was born August 29, 1958 – June 25, 2009
+    Nicknamed the "King of Pop", he is regarded as one of the most significant cultural figures of the 20th century.</li>
+   <li>Over a four-decade career, his contributions to music, dance, and fashion, 
+along with his publicized personal life, made him a global figure in popular culture.</li>
+   <li>Jackson influenced artists across many music genres; through stage and video performances.</li>
+    <li>he popularized complicated dance moves such as the moonwalk, to which he gave the name, as well as the robot.</li>
+     <li> He is the most awarded individual music artist in history.</li>
+      </ul> </p>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <script  src="js/index.js"></script>
+  </body>
+  <a id="tribute-link" href="https://en.wikipedia.org/wiki/Michael_Jackson" target="_blank"> Learn More.</a> <br>
+<br>
+  </main>
+</html>
```

Comment:
Design for this project is very minimal. Please try to add some features and styling.